tasks/cluster.py

```diff
@@ -57,9 +53,10 @@ def provision(ctx):
         [
             "--name {}".format(AKS_CLUSTER_NAME),
             "--node-count {}".format(AKS_CLUSTER_NODE_COUNT),
-            "--node-vm-size {}".format(AZURE_VM_SIZE),
+            "--node-vm-size {}".format(AZURE_K8S_VM_SIZE),
+            "--os-sku Ubuntu",
```

Comment:
I never worried about setting this parameter, did you find it necessary?

Reply:
It's not necessary as I think it's the default, but I couldn't work out what OS we were using and wanted to be explicit about Ubuntu in the hope that I could put other SKUs in here. We need to have at least 20.04, preferably with HWE installed, so I want to fiddle with this a bit more in future.